src/Rector/Deprecation/AssertNoUniqueTextRector.php

```diff
@@ -17,11 +17,6 @@ final class AssertNoUniqueTextRector extends AbstractRector
 
     use GetDeclaringSourceTrait;
 
-    /**
-     * @todo remove when property is no longer private in AbstractRector.
-     */
-    private $nodesToAddCollector;
-
```

Comment:
Keep but make protected since change is not released.

Reply:
Done